### DISCUSSION TITLE
no usage updates last couple of days

### DISCUSSION THREAD

**Discussion by icenov** [amberelectric/public-api#109] [General]
Hi
I haven't had a usage update since Sunday. Tried logout/login but no change.
Has something changed?
Tony

**Comment by joho** [comment]
A bug in our data pipelines. We fixed the root cause this morning and are in the process of backfilling all the gaps now.

**Reply by oldwindmill** [reply]
missing again? nothing for sat, tues or wed....but considering the current prices, maybe its better that way....

**Reply by oldwindmill** [reply]
or thursday

**Reply by loadx** [reply]
Hi @oldwindmill i'd like to have our dev team review this. Can you please drop a support ticket via email to info@amber.com.au (this helps us to identify your account) and we'll be in touch as soon as we can.

Thanks & have a great weekend :)

**Reply by oldwindmill** [reply]
email sent. didnt see your request till just now..
